Plugins/NativeEngine/Source/NativeEngine.h

```diff
@@ -234,7 +235,7 @@ namespace Babylon
     {
         FrameBufferManager()
         {
-            m_boundFrameBuffer = m_backBuffer = new FrameBufferData(BGFX_INVALID_HANDLE, GetNewViewId(), bgfx::getStats()->width, bgfx::getStats()->height);
+            m_boundFrameBuffer = m_defaultBackBuffer = new FrameBufferData(BGFX_INVALID_HANDLE, 0, bgfx::getStats()->width, bgfx::getStats()->height);
```

Comment:
Is this a behavior change from the way it worked before? If so, do we know why it used to be different?